_studio/mfx_lib/decode/mjpeg/src/mfx_mjpeg_dec_decode.cpp

```diff
@@ -1025,6 +1020,7 @@ bool MFX_JPEG_Utility::IsNeedPartialAcceleration(VideoCORE * core, mfxVideoParam
 
     if (core->GetHWType() == MFX_HW_APL
```

Comment:
Shouldn't we re-phrase this one as type == APL || type >= KBL ? I.e. cover ICL and future platforms.

Reply:
+1

Reply:
@onabiull @dvrogozh Done.